reproduce.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+venv_dir=".anomaly"
```

Comment:
I think pull this out and have a setup.sh and a run.sh. Recommend persons use a venv, "use setup.sh if you want", but otherwise just provide something people can run to make results

Reply:
Sounds right. Let’s get something up before we get shot down for lack of reproducibility.